Sources/PagerTabItemModifier.swift

```diff
@@ -24,9 +24,8 @@ struct PagerTabItemModifier<NavTabView: View> : ViewModifier {
                         index = Int(round(frame.minX / settings.width))
                         let tabView = navTabView()
                         let tabViewDelegate = tabView as? PagerTabViewDelegate
-                        dataStore.setView(AnyView(tabView),
```

Comment:
Separating this in 2 looks good at first but what about all the views listening for changes to this environment object? Now you will have 2 updates everywhere, right?

Reply:
Looks like we only use the count var and have only published vars on the collection and count as shown below . 

```
@Published var items = [Int: DataItem](){
        didSet {
            itemsCount = items.count
        }
    }

    @Published private(set) var itemsCount: Int = 0
 ```